docs/README.md

```diff
@@ -2,19 +2,19 @@
 
 ## Overview
```

Comment:
Is a table of contents suitable here? If so, that might help the readability of this lengthy page. Suggestion, not a requirement :)

Reply:
This file is going to be deprecated(following PR) the reason I made the changes here is for alignment and reference (I also used it to compare between website -User Guide).